lib/KmsClient.js

```diff
@@ -789,15 +789,15 @@ function _handleClientError({
     error = new Error(`${errorMessage}: ${notFoundMessage}`);
     error.status = 404;
   } else {
-    error = cause;
-    error.message = `WebKMS client error: ${cause.message}`;
+    error = new Error(`WebKMS client error: ${errorMessage}`);
   }
```

Comment:
I recommend we do this for some backwards compatibility as well:

```suggestion
    error = new Error(`WebKMS client error: ${errorMessage}`);
    error.status = cause.status;
  }
```

Reply:
@dlongley do you think we should test that `cause.status` exists? to avoid `error.status = undefined`.

Reply:
Sure, let's add that too.